bower.json

```diff
@@ -24,8 +24,11 @@
   "license": "Apache-2.0",
   "homepage": "https://googlewebcomponents.github.io/google-youtube",
   "dependencies": {
-    "core-localstorage": "Polymer/core-localstorage#^0.5.4",
-    "google-apis": "GoogleWebComponents/google-apis#~0.4.3",
-    "polymer": "Polymer/polymer#^0.5.4"
+    "iron-localstorage": "PolymerElements/iron-localstorage#0.8-preview",
+    "google-apis": "GoogleWebComponents/google-apis#0.8-preview",
+    "polymer": "Polymer/polymer#0-8.preview"
```

Comment:
The import will be bad. The bower path for polymer would be listed as:
"polymer": "Polymer/polymer#0.8-preview"